metals/src/main/scala/scala/meta/internal/metals/config/RunType.scala

```diff
@@ -3,12 +3,14 @@ package scala.meta.internal.metals.config
 object RunType {
   sealed trait RunType
   case object Run extends RunType
+  case object RunFile extends RunType
```

Comment:
This is just a proposal, but I think this would be useful in a lot of different clients.

Reply:
Do you think we should match the name with `RunOrTestFile`? No strong feeling though

Reply:
Forgot about it! Fixed!